wiki/Handling_Cheating/pt-br.md

```diff
@@ -0,0 +1,75 @@
+Por que fui banido?
+----------------
+
+-   10 silêncios emitidos na sua página de usuário.
+-   Você deu a sua conta para outra pessoa.
+-   Usou várias contas.
+-   Trapaceou (veja abaixo).
+
+O que é considerado trapaça?
+----------------------------
+
+Como as regras dizem, qualquer forma de manipulação do cliente; hack de pontuação, macro em sequencias de combo e mira automática são todas consideradas trapaça.
+
+Isso é uma regra de um golpe; trapaceiros serão **banidos permanentemente.**
```

Comment:
Not sure if "regra de um golpe" is nice to read. What about "regra única" ou "regra imutável"?